R/mrmash_wrapper.R

```diff
@@ -297,6 +283,7 @@ filter_data_driven_mats <- function(Y, data_driven_mats) {
 
   # Check if colnames of Y is a subset of column names of each element in data_driven_mats
   for (mat_name in names(data_driven_mats)) {
+    if (mat_name == "null") data_driven_mats[[mat_name]] <- NULL; next # remove null component 
```

Comment:
this is not a good check. I think you should check if all element is zero for the null component?